client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.html

```diff
@@ -0,0 +1,102 @@
+<div id="cloud-project-compute-loadbalancer" class="row">
+    <div class="cui-page__content">
+
+        <div ng-if="!CloudProjectComputeLoadbalancerCtrl.loaders.table.loadbalancer && CloudProjectComputeLoadbalancerCtrl.table.loadbalancer && !CloudProjectComputeLoadbalancerCtrl.table.loadbalancer.length"
+             class="oui-box">
+              <h1>Load Balancers</h1>
+              <p>
+                TODO : include this https://www.ovh.com/fr/solutions/load-balancer/
+              </p>
+        </div>
+
+        <div ng-if="CloudProjectComputeLoadbalancerCtrl.loaders.table.loadbalancer"
+             class="text-center">
+            <oui-loader size="m" class="d-inline-block"></oui-loader>
+        </div>
+
+        <div ng-if="CloudProjectComputeLoadbalancerCtrl.table.loadbalancer && CloudProjectComputeLoadbalancerCtrl.table.loadbalancer.length">
+            <a
+                type="button"
+                class="oui-button oui-button_secondary"
+                data-ng-disabled="CloudProjectComputeLoadbalancerCtrl.loaders.table.loadbalancer"
+                ng-href="{{CloudProjectComputeLoadbalancerCtrl.urls.website_order.load_balancer[CloudProjectComputeLoadbalancerCtrl.locale]}}">
+                <i class="glyphicon glyphicon-plus"></i>
+                <span data-translate="cpc_loadbalancer_order"></span>
+            </a>
+
+            <!-- Loadbalancer TABLE -->
+            <table class="oui-table oui-table_responsive">
```

Comment:
oui-table.

Reply:
Use directly the oui-table component : https://github.com/ovh-ux/ovh-ui-angular

Reply:
<!-- Loadbalancer TABLE --> => useless comment.  People should know how to read code and complex code should be simplified ;)